Tests/PointFreeTests/__Snapshots__/BlogTests/testBlogIndex.1.Conn.txt

```diff
@@ -10,10 +10,10 @@
 
 ▿ Response
   Status 200 OK
-  Content-Length: 45010
+  Content-Length: 100724
```

Comment:
you can see just how much the blog index page is increasing here.

p cool that we have a history of the file size of all our pages.